packages/lib/src/vis/shared/PanMesh.tsx

```diff
@@ -17,14 +16,15 @@ function PanMesh(props: Props) {
   const { disabled, modifierKey } = props;
 
   const camera = useThree((state) => state.camera);
+  const size = useThree((state) => state.size);
 
   const startOffsetPosition = useRef<Vector3>(); // `useRef` to avoid re-renders
 
   const moveCameraTo = useMoveCameraTo();
 
   const onPointerDown = useCallback(
-    (evt: ThreeEvent<PointerEvent>) => {
-      const { sourceEvent, unprojectedPoint } = evt;
+    (sourceEvent: PointerEvent) => {
+      const unprojectedPoint = getUnprojectedPoint(sourceEvent, size, camera);
```

Comment:
`unprojectedPoint` is computed through a function that takes the `size` and the `camera` as arg. 

We could create a callback out of a hook to avoid passing such arg but most callers already get the `camera` via `useThree`. And I am not convinced about the need for `size` (see the discussion on `getUnprojectedPoint` below)